wiki/Ranking_Criteria/osu!mania/es.md

```diff
@@ -0,0 +1,205 @@
+# Criterios de rankeo de osu!mania
+
+Los **Cirterios de rankeo de osu!mania** son reglas y guías que aplican a la creación de dificultades especificas a osu!mania. Con el fin de rankear una dificultad especifica de osu!mania, es mandatorio que la creación de la misma obedezca los criterios listados. Mientras que **todas las reglas deben de seguirse en cualquier circunstancia**, las guías pueden ser ignoradas durante circunstancias excepcionales. Estas circunstancias excepcionales deben ser justificadas por una explicación exhaustiva de porque la guía fue ignorada y como interfiere con la calidad en general de la creación.
+
+Propuestas de reglas también como sugerencias son discutidas en [Ranking Criteria Subforum](https://osu.ppy.sh/community/forums/87). Cualquier regla que paso por una discusión y aprobación de la comunidad están listadas aquí como han sido puestas en su hilo de discusión designado.

```

Comment:
```suggestion
Este conjunto de **criterios de clasificación de osu!mania** establece [reglas y pautas](/wiki/Ranking_Criteria#términos_generales) que los [beatmaps](/wiki/Beatmap) específicos de [osu!mania](/wiki/Game_mode/osu!mania) deben seguir para avanzar en el [procedimiento de clasificación de beatmaps](/wiki/Beatmap_ranking_procedure).
```

Reply:
I don't think that line should be deleted with that new one, it already states in line 3 that this are the rules and guidelines needed to push a map into the ranked section.